underfs/wasb/src/main/java/alluxio/underfs/wasb/WasbUnderFileSystem.java

```diff
@@ -36,15 +38,18 @@
   private static final Logger LOG = LoggerFactory.getLogger(WasbUnderFileSystem.class);
 
   /** Constant for the wasb URI scheme. */
-  public static final String SCHEME = "wasb://";
+  public static final List<String> SCHEMES = Stream.of("wasb://", "wasbs://")
+          .collect(Collectors.toList());
 
   /**
    * Prepares the configuration for this Wasb as an HDFS configuration.
    *
    * @param conf the configuration for this UFS
+   * @param isWasb whether to use wasb configuration or not
    * @return the created configuration
    */
-  public static Configuration createConfiguration(UnderFileSystemConfiguration conf) {
+  public static Configuration createConfiguration(UnderFileSystemConfiguration conf,
+          Boolean isWasb) {
```

Comment:
`isWasb` doesn't seem very clear to me. Do you think something like `isSecure` might be easier to understand? - If so, you'd also have to flip the logic for the conditional below

Reply:
Sure, I have changed this flag to `isSecure`.